tests/simulation/simulation_test.go

```diff
@@ -418,22 +388,62 @@ func toBlock(t *testing.T, simBlock SimulationBlock) block.Block {
 	return b
 }
 
+func submitTickets(
+	t *testing.T,
+	keys []ValidatorKeys,
+	currentState *state.State,
+	currentTimeslot jamtime.Timeslot,
+	ticketAttempts map[string]int,
+) []block.TicketProof {
+	nextEpoch := currentTimeslot.ToEpoch()
+	previousEpoch := currentState.TimeslotIndex.ToEpoch()
+
+	entropy := currentState.EntropyPool[2]
+	pendingValidators := currentState.ValidatorState.SafroleState.NextValidators
+	if nextEpoch > previousEpoch {
+		pendingValidators = validator.NullifyOffenders(currentState.ValidatorState.QueuedValidators, currentState.PastJudgements.OffendingValidators)
+		entropy = currentState.EntropyPool[1]
+	}
+
+	ticketProofs := []block.TicketProof{}
+	for _, key := range keys {
+		if ticketAttempts[key.Name] < common.MaxTicketAttempts {
+			attempt := ticketAttempts[key.Name]
+			ticketProducerKey := crypto.BandersnatchPrivateKey(testutils.MustFromHex(t, key.BandersnatchPrivate))
+			// TODO this will need fancier logic too. Needs to use the right yk and eta depending on epoch change.
+			ticketProof, err := state.CreateTicketProof(pendingValidators, entropy, ticketProducerKey, uint8(attempt))
+			require.NoError(t, err)
+			t.Logf("submitted ticket, name: %v, attempt: %v, proof: %v", key.Name, attempt,
+				hex.EncodeToString(ticketProof.Proof[:])[:10]+"...")
+			ticketProofs = append(ticketProofs, ticketProof)
+			ticketAttempts[key.Name]++
+		}
+		if len(ticketProofs) == common.MaxTicketExtrinsicSize {
+			break
+		}
+	}
+
+	return ticketProofs
+}
+
 // Only these fields are needed for now. Extrinsics will be added as we need them.
 type SimulationBlock struct {
-	Header struct {
-		Parent          string `json:"parent"`
-		ParentStateRoot string `json:"parent_state_root"`
-		ExtrinsicHash   string `json:"extrinsic_hash"`
-		Slot            int    `json:"slot"`
-		EpochMark       *struct {
-			Entropy        string   `json:"entropy"`
-			TicketsEntropy string   `json:"tickets_entropy"`
-			Validators     []string `json:"validators"`
-		} `json:"epoch_mark"`
-		AuthorIndex   int    `json:"author_index"`
-		EntropySource string `json:"entropy_source"`
-		Seal          string `json:"seal"`
-	} `json:"header"`
+	Header `json:"header"`
```

Comment:
Is there a reason for this change right now? I think I can just fix this up once your PR is in and switch the sim test to use the new block encoding in jsonutils.